src/Data/Unfoldable1.purs

```diff
@@ -103,6 +123,8 @@ singleton = replicate1 1
 -- | range 0 0 == (NEL.singleton 0 :: NEL.NonEmptyList Int)
 -- | range 1 2 == (NEL.cons 1 (NEL.singleton 2) :: NEL.NonEmptyList Int)
 -- | range 2 0 == (NEL.cons 2 (NEL.cons 1 (NEL.singleton 0)) :: NEL.NonEmptyList Int)
```

Comment:
Writing the Non-empty examples is kinda clumsy without https://github.com/purescript/purescript-lists/pull/190